modules/core/src/test/resources/config/default.properties

```diff
@@ -2,5 +2,6 @@ scan.path=/tmp/test
 scan.identifier.regex=(.*?)_.*
 scan.tags.createFromFolder=true
 filter.tags.blacklisted=un,deux
+tags=TOTO=[src/test/resources/scanner_test_files/01_file1;src/test/resources/scanner_test_files/subfolder/*],potato=[src/test/resources/scanner_test_files/*;src/test/resources/scanner_test_files/subfolder/03_file3]
```

Comment:
We didn't speak about the way to configure tags. I guess we should :-)

I would prefer :
- one property line per tag
- glob path matchers list with comma as separator

Example:
`
tags.TOTO=[src/test/resources/scanner_test_files/01_file1,src/test/resources/scanner_test_files/subfolder/*]
tags.potato=[src/test/resources/scanner_test_files/*,src/test/resources/scanner_test_files/subfolder/03_file3]
`

Don't you find it is more readable ?
Same remark for the CLI. `--tag TOTO=[] --tag potato=[]`

Free to speak with you about that.